### DISCUSSION TITLE
[游戏推荐]米哈游新作，绝区零公测啦！！！

### DISCUSSION THREAD

**Discussion by RETRN1** [share121/inter-knot#1688] [其他]
![zzz bt](https://github.com/user-attachments/assets/7f29f250-9b8f-46ac-8929-a570c485e048)

![37198ce9c5ee13abb2c49f1bd1c3ca97_7702488080246216541](https://github.com/user-attachments/assets/0473f997-ee28-4254-9474-49630028e924)

![play](https://github.com/user-attachments/assets/d724840a-650b-4926-b582-6b903ee677eb)

![zzz](https://github.com/user-attachments/assets/aaf6609e-b566-4ac1-a212-9e2b8e344ca3)

# 官网：
https://zzz.mihoyo.com/main/
# 下载：
https://autopatchcn.juequling.com/package_download/op/client_app/download/20240618183950_ehxltROsR67s5q43/gwpc/ZenlessZoneZero_setup_202406181742.exe
# 介绍：
应用名称：绝区零 开发者名称：上海交宅科技有限公司
健康游戏忠告：抵制不良游戏，拒绝盗版游戏。注意自我保护，谨防受骗上当。适度游戏益脑，沉迷游戏伤身。合理安排时间，享受健康生活。
© 2022 米哈游版权所有 | 上海米哈游影铁科技有限公司

**Comment by lllliufeng** [comment]
我来啦

**Comment by gaojiaobei** [comment]
是全新的动作游戏！听说是以咱们现在这个时间为背景的现实题材游戏，这下真的很期待了

**Comment by Xieleng42** [comment]
玩了，真是太好玩了！特别是里面的游戏机，简直一模一样！

**Comment by tuzibuqiahuluobo** [comment]
新艾利都啥时候有这家公司了？

**Comment by hanshui2077** [comment]
很好玩，里面有个高仿绳网，听说和这个很像（doge
